lang/sql/sql.py

```diff
@@ -5,6 +5,10 @@
 tag: user.sql
 """
 
+ctx.lists["user.code_data_null"] = {
+    "null": "NULL",
```

Comment:
This drops "nil", "none", and "no". I do find it useful not to need to think about whether a language uses "none" or "null" when switching back and forth. I wonder if there's a way to recover that behaviour without a bunch of duplication 🤔

Reply:
Then what you want is probably an action? Because then you could have a bunch of different spoken phrases that would just insert a correct thing for the language.

Reply:
Are you suggesting an action `code_insert_null(kind: str)`? I suppose that could work.

Reply:
I was thinking more that was pokey liked was the action you just removed, but maybe a middle ground can be reached?

Reply:
The entire premise of this PR is being able to get _different_ things for `none` and `null` in languages where the two exist, are similar, but different concepts. I don't see how we can achieve that without a list.

Reply:
Me neither.

Reply:
I just came up with a wild idea as an alternative to adding a `code_insert_null` action, which is potentially overengineering it, but here it is:

- We wrap the list in a capture.
- To simplify implementing languages with a single null-like value, we add a str setting to allow a language to indicate it has a single null-like value. The capture would check this setting, and if unset it passes through the list value as is.

I'm not sure whether I like wrapping it in an action, or a capture more.